### PR TITLE
SV: Add missing languages

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

File: web/app/configs/lang.json [hunk 1]
```diff
@@ -801,16 +801,18 @@
...
+            "AD": "Andorra",
             "AL": "Albanien",
             "AM": "Armenien",
             "AT": "Österrike",
+            "AR": "Argentina",
             "AUS-NSW": "Australien (New South Wales)",
             "AUS-NT": "Australien (Northern Territory)",
             "AUS-QLD": "Australien (Queensland)",
-            "AUS-SA": "Australien (South Australien)",
+            "AUS-SA": "Australien (South Australia)",
```
Comment (corradio) [inline web/app/configs/lang.json]: Shouldn't it be "Södra australien"?

**Comment by magol** [comment]
@corradio I have been unsure whether to translate this. When I look around the internet, some use the English name, and others a translated name.

To know how to do, I wrote to the Australian Embassy in Sweden and asked what they recommended. And they say i should use the English name.

File: web/app/configs/lang.json (hunk 1 above)
Comment (magol) [reply web/app/configs/lang.json]: @corradio I have been unsure whether to translate this. When I look around the internet, some use the English name, and others a translated name.

To know how to do, I wrote to the Australian Embassy in Sweden and asked what they recommended. And they say i should use the English name.

Comment (corradio) [reply web/app/configs/lang.json]: 👍

**Pull request merged**